documentation/src/main/asciidoc/topics/proc_using_jgroups_default.adoc

```diff
@@ -0,0 +1,57 @@
+[id='jgroups_getting_started-{context}']
+= Getting Started with Default Stacks
+{brandname} uses JGroups protocol stacks so nodes can send each other messages on dedicated cluster channels.
+
+{brandname} provides preconfigured JGroups stacks for `UDP` and `TCP` protocols.
+You can use these default stacks as a starting point for building custom cluster transport configuration that is optimized for your network requirements.
+
+.Procedure
+
+. Locate the default JGroups stacks, `default-jgroups-*.xml`, in the `default-configs` directory inside the `infinispan-core-{ispn_version}.jar` file.
```

Comment:
would be a generic ispn-version better ?

Reply:
This resolves to the actual version when the docs are built. It's tied into the CI stuff.